locales/ro/LC_MESSAGES/volto.po

```diff
@@ -0,0 +1,1939 @@
+
+#: components/theme/Login/Login
+msgid "If you have forgotten your password, {forgotpassword}"
+msgstr "Dacă v-ați uitat parola, {forgotpassword}"
+
+#: components/theme/Login/Login
+msgid "If you you do not have an account here, head over to the {registrationform}."
+msgstr "Dacă nu aveți un cont aici, treceți la {registrationform}."
+
+#: components/manage/Blocks/Image/ImageSidebar
+#: components/manage/Blocks/LeadImage/LeadImageSidebar
+msgid "Image"
+msgstr "Imagine"
+
+#: components/manage/Controlpanels/ContentType
+#: components/manage/Controlpanels/Controlpanel
+msgid "Info"
+msgstr "Info"
+
+#: components/manage/Sharing/Sharing
+msgid "Inherit permissions from higher levels"
+msgstr "Permisiuni mostenite de la niveluri superioare"
+
+#: components/manage/Sharing/Sharing
+msgid "Inherited value"
+msgstr "Valoarea moștenită"
+
+#: components/manage/Blocks/Table/Edit
+msgid "Insert col after"
+msgstr "Introduceți col. după"
+
+#: components/manage/Blocks/Table/Edit
+msgid "Insert col before"
+msgstr "Introduceți col. înainte"
+
+#: components/manage/Blocks/Table/Edit
+msgid "Insert row after"
+msgstr "Introduceți rândul după"
+
+#: components/manage/Blocks/Table/Edit
+msgid "Insert row before"
+msgstr "Introduceți rândul înainte"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Install"
+msgstr "Instalare"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Installed"
+msgstr "Instalat"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Installed version"
+msgstr "Versiunea instalată"
+
+#: components/manage/Blocks/Listing/ListingData
+msgid "Item batch size"
+msgstr "Dimensiunea lotului obiectului"
+
+#: components/manage/Contents/Contents
+msgid "Item(s) copied."
+msgstr "Obiect(e) copiat."
+
+#: components/manage/Contents/Contents
+msgid "Item(s) cut."
+msgstr "Obiect(e) tăiat(e)."
+
+#: components/manage/Actions/Actions
+#: components/manage/Contents/Contents
+msgid "Item(s) pasted."
+msgstr "Obiect(e) lipit(e)."
+
+#: components/manage/Controlpanels/ContentTypes
+msgid "Items"
+msgstr "Obiecte"
+
+#: components/manage/Form/Form
+#: components/manage/Form/ModalForm
+msgid "Items must be unique."
+msgstr "Obiectele trebuie să fie unice."
+
+#: components/manage/Preferences/PersonalPreferences
+msgid "Language"
+msgstr "Limba"
+
+#: components/manage/Contents/Contents
+msgid "Last comment date"
+msgstr "Data ultimului comentariu"
+
+#: components/manage/Contents/ContentsUploadModal
+msgid "Last modified"
+msgstr "Ultima modificare"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Latest version"
+msgstr "Ultima versiune"
+
+#: components/manage/Blocks/LeadImage/LeadImageSidebar
+msgid "Lead Image"
+msgstr "Imaginea de start"
+
+#: components/manage/Blocks/Maps/Edit
+#: helpers/AlignBlock/AlignBlock
+msgid "Left"
+msgstr "Stânga"
+
+#: components/manage/Toolbar/Toolbar
+msgid "Link"
+msgstr "Link"
+
+#: components/manage/Blocks/Listing/ListingSidebar
+msgid "Link more"
+msgstr ""
```

Comment:
I think it's about a link for more items

Reply:
Fixed.